src/components/NavbarTopNotif.js

```diff
@@ -1,14 +1,17 @@
-//TODO: create component search bar
+//TODO: create component notif
 
 import React from "react"
 import { NavItem } from "reactstrap"
 import NavbarTopNotifBubble from "./NavbarTopNotifBubble"
 
-const NavbarTopNotif = () => (
-  <NavItem>
-    <img src={process.env.PUBLIC_URL + "img/iconNotif.ico"} alt="iconNotif" />
-    <NavbarTopNotifBubble />
-  </NavItem>
-)
+const NavbarTopNotif = ({ displayNumberNotifs }) => {
+  console.log(displayNumberNotifs)
+  return (
+    <NavItem>
+      <img src={process.env.PUBLIC_URL + "img/iconNotif.ico"} alt="iconNotif" />
+      <NavbarTopNotifBubble numberNotifs={displayNumberNotifs} />
```

Comment:
please be consistent with props name, you can use the same name across many components
numberNotifs (notifsCount is a better name by the way)
use the same name everywhere